sdk/JavaScript/README.md

```diff
@@ -0,0 +1 @@
+The javascript tests for the MobileApp branch now live at: https://github.com/Azure/azure-mobile-apps-js-client

```

Comment:
We removed managed and iOS from /sdk directory without adding any README files. For consistency purpose, we should either add README for managed and iOS, or not adding this README for Javascript. :laughing: